spec/calendar_spec.rb

```diff
@@ -2,6 +2,12 @@
 
 describe Icalendar::Calendar do
 
+  context 'marshalling' do
+    it 'can be de/serialized' do
+      Marshal.load(Marshal.dump(subject))
```

Comment:
Is this a strong enough test that this is working properly?

Reply:
I've been using a patched version since creating this PR and I haven't found any calendars that would break it so so far it's good enough for me. If anyone finds new ways to break it will have to be extended.